AngularPart1/Components/Home/home.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="toDoController">
+    <h3>Active ({{ tasks.length }})</h3>
+    <span ng-if="tasks.length === 0">You have no active tasks!</span>
+    <div class="filters">
+        <input type="text" ng-model="filterObj.letters" ng-change="filterByLetters()">
```

Comment:
Please reuse appropriate built-in API for filtering https://docs.angularjs.org/api/ng/filter/filter